package.json

```diff
@@ -36,7 +36,7 @@
     "test:debug": "mocha debug test/runner.js --reporter spec"
   },
   "dependencies": {
-    "broccoli-caching-writer": "^0.5.4",
+    "broccoli-caching-writer": "^1.1.0",
```

Comment:
the root problem is ember-cli, providing extremely vague inputTrees resulting in way to many inputFiles being considered/processed for the cache key.

We will need to break that part of ember-cli likely, but it will finally fix the underlying issue.